container/navigation.go

```diff
@@ -0,0 +1,207 @@
+package container
+
+import (
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/layout"
+	"fyne.io/fyne/v2/theme"
+	"fyne.io/fyne/v2/widget"
+)
+
+// Navigation container is used to provide your application with a control bar and an area for content objects.
+// Objects can be any CanvasObject, and only the most recent one will be visible.
+//
+// Since: 2.7
+type Navigation struct {
+	widget.BaseWidget
+	level  int
```

Comment:
Can the private fields be after the public ones? at least for initial commit of a widget that fits our ordering better